apps/client/src/components/molecules/button/Button.component.tsx

```diff
@@ -0,0 +1,72 @@
+import { ReactElement } from 'react';
+import { Text } from '../../';
+import { ReactComponent as CarIcon } from '../../../assets/cars/pixelCar.svg';
+import cs from 'classnames';
+
+export interface ButtonProps {
+  onClick: () => void;
+  isCarVisible?: boolean;
+  isDisable?: boolean;
+  isButtonDanger?: boolean;
+  children: string;
+}
+
+export function Button({
+  onClick,
+  isCarVisible,
+  isDisable,
+  isButtonDanger,
+  children,
+}: ButtonProps): ReactElement {
+  isCarVisible ||= false;
+  isDisable ||= false;
+  isButtonDanger ||= false;
```

Comment:
Why are we doing like this instead of destructuring ?

Reply:
changed a265db1e6bf1a787fd82cac9340b3925986267ce in PR #52